src/Data/Conduit/Tar.hs

```diff
@@ -88,7 +88,7 @@ headerFileType h =
     case headerLinkIndicator h of
         0  -> FTNormal
         48 -> FTNormal
-        49 -> FTHardLink
+        49 -> FTHardLink (fromShort (headerLinkName h))
```

Comment:
That's the right way to go about it, but unfortunately that's a breaking change to the API, so we'll have to do a major version bump

Reply:
Is your policy like @snoyberg's that you'd like @gaborh-da to bump the .cabal? Or like @ndmitchell's that you prefer to do that when the release is made? I'm sure either can be accommodated.

Reply:
To be honest, I have no preference :) I can bump up the version and make the release, it's no trouble for me.

Reply:
To be honest, I would prefer if you did it. :)